Misc/NEWS

```diff
@@ -806,6 +806,8 @@ Documentation
 Build
 -----
 
+- Issue #29572: Update Windows build and OS X installers to use OpenSSL 1.0.2k.
```

Comment:
I think the change here only reflect Windows build.
Is there going to be a separate PR for OS X?